Automattic/jetpack
# Jetpack: improve forms inputs layout

This PR improves the form layout by removing the input's right-border (used for spacing purposes) and refactor a bit the layout and innerblocks flex directives.

This makes for a more consistent handling of blocks (with no dangling borders/margins). The use of `calc` might not be _pixel perfect_ but it will hold for the functionality the wrappers are offering.

Also, we're using CSS var `--wp--style--block-gap` and defaulting to `1.5rem` if it's not set.

This is a "first step" as we seek to fully address lack of consistency on some of the form's UX

#### Changes proposed in this Pull Request:

* Remove the border-right directive on all inputs/wrappers
* Add `gap` on the block/form wrapper layout
* Change flex directives to flex-basis X% and limit width with max-width on all 25, 50 and 75 wrappers

#### Other information:


#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
Insert a form and play with the inputs widths at the sidebar settings.
- [ ] inputs should reach all the way to the right limit of the block editor (when they should, of course)
- [ ] inputs should have a gap between them when put side by side

Publish and visit the post, form layout should be consistent with the editor
